webapp/application.py

```diff
@@ -484,6 +486,90 @@ def application_withdrawal(token):
         f"application_id={payload['application_id']}"
     )
 
+    # get candidate interviews from Harvest API and filter
+    candidate_interviews = harvest.get_interviews_scheduled(application["id"])
+    candidate_interviews = [
+        interview
+        for interview in candidate_interviews
+        if interview["status"] in ["scheduled", "awaiting_feedback"]
+    ]
+    all_sent_emails = []
+
+    calendar = CalendarAPI()
+    for interview in candidate_interviews:
+        # get interviewer information
+        interviewer = interview["interviewers"][0]
+        interviewer_timezone = calendar.get_timezone(interviewer["email"])
+
+        # convert interview time to interviewer's timezone
+        date_time_str = interview["start"]["date_time"]
+        date_time_obj = datetime.fromisoformat(
+            date_time_str.replace("Z", "+00:00")
+        )
+        date_time_obj = date_time_obj.astimezone(
+            pytz.timezone(interviewer_timezone)
+        )
```

Comment:
instead of `date_time_str` and `date_time_str` lets name it to `interview_start_time_str` so we can stick to verbose variables names as much as possible